client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEdit.tsx

```diff
@@ -36,24 +32,22 @@ import { PrescriptionLineEditForm } from './PrescriptionLineEditForm';
 import { PrescriptionLineEditTable } from './PrescriptionLineEditTable';
 import { ItemRowFragment } from '@openmsupply-client/system';
 import { usePrescriptionLines } from '../../api/hooks/usePrescriptionLines';
+import { Footer } from './Footer';
 
 interface PrescriptionLineEditModalProps {
-  isOpen: boolean;
-  onClose: () => void;
   draft: Draft | null;
   mode: ModalMode | null;
+  items: ItemRowFragment[];
 }
 
 export const PrescriptionLineEdit: React.FC<PrescriptionLineEditModalProps> = ({
```

Comment:
Since this is no longer a modal, we probably want to do some renaming to remove all references to "Modal" (including the "mode" prop values)